vhdbuilder/packer/kube-proxy-images.json

```diff
@@ -12,13 +12,16 @@
           "1.23.12-hotfix.20221109.2",
           "1.23.8-hotfix.20221109.1",
           "1.23.8-hotfix.20221109.2",
+          "1.23.15",
```

Comment:
kube-proxy for k8s 1.23 patch versions, needs to use the hotfix version (after creating separate build)

Reply:
Curious: 1.23.15 is the latest patch version for v1.23. why do we still need to hotfix it? https://github.com/kubernetes/kubernetes/releases

Reply:
It's because this feature is needed (Detect local traffic using interface): https://github.com/kubernetes/kubernetes/pull/95400, in k8s 1.23 on AKS as well (upstream has it merged for k8s 1.24+).

Reply:
New PR here: https://github.com/Azure/AgentBaker/pull/2574